rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -159,18 +164,14 @@ def _find_trades(self, all_logs: list[EvmTxReceiptLog]) -> list[CowswapSwapData]
                 owner_address == self.native_asset_flow_address
             ):
                 from_asset = self.native_asset  # native asset swaps are made by eth flow contract in cowswap  # noqa: E501
-            else:  # token should exist since this is the post-processing stage
-                from_asset = EvmToken(evm_address_to_identifier(
+            else:
```

Comment:
Add a comment here, negating the previous comment. Explain why this token may not exist here.
```suggestion
            else:  # need get_or_create_evm_token because token may not exist WHEN?
```

Is the answer batched trades? I don't really get the situation. So an explanation is needed here